config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
+  # config.default_per_page = 25
```

Comment:
Nope. You can generate the config file later but I thought better to have something you can grep for.